webpack.config.js

```diff
@@ -12,7 +12,8 @@ const fs = require( 'fs' );
 const path = require( 'path' );
 const webpack = require( 'webpack' );
 const AssetsWriter = require( './server/bundler/assets-writer' );
-const MiniCssExtractPlugin = require( 'mini-css-extract-plugin' );
+const MiniCssExtractPlugin = require( 'mini-css-extract-plugin-with-rtl' );
```

Comment:
Should we rename the import to reflect the `WithRtl` part?

Reply:
Hmm, I'd rather keep the name without a suffix. It's a forked plugin that will likely be merged into upstream sooner or later.

Reply:
We might need to actively lobby the upstream project for this to happen tho...

Reply:
> We might need to actively lobby the upstream project for this to happen tho...

I just got involved by [commenting on an issue](https://github.com/webpack-contrib/mini-css-extract-plugin/issues/217#issuecomment-416137283) and by filing another, [somewhat related one](https://github.com/webpack-contrib/mini-css-extract-plugin/issues/253).